ocs_ci/helpers/sanity_helpers.py

```diff
@@ -183,3 +182,156 @@ def __init__(self):
         self.pod_objs = list()
         self.obc_objs = list()
         self.ceph_cluster = CephClusterExternal()
+
+
+class SanityManagedService(Sanity):
+    """
+    Class for cluster health and functional validations for the Managed Service
+    """
+
+    def __init__(self):
+        """
+        Init the sanity managed service class
+        """
+        super(Sanity, self).__init__()
```

Comment:
could be that you will need to replace with:
```python
super(SanityManagedService, self).__init__()
```

Reply:
Yes, fixed.